training/components.py

```diff
@@ -85,17 +85,29 @@ def pytorchjob_manifest_op(
     input_pvc_name: str,
     output_pvc_name: str,
     name_suffix: str,
-    path_to_model: str,
+    # path_to_model: str,
     phase_name: str,
     nproc_per_node: int = 3,
     nnodes: int = 2,
 ) -> NamedTuple("outputs", manifest=str, name=str):
     import inspect
+    import os
+
+    def list_phase1_final_model():
+        model_dir = "/output/model/hf_format"
+        models = os.listdir(model_dir)
+        newest_idx = max([(int(j.split("_")[-1]), i) for i, j in enumerate(models)])[-1]
```

Comment:
hum, how about we use `mtime` to sort?

```suggestion
        newest_idx = max((os.path.getmtime(model), i) for i, model in enumerate(models))[-1]
```

Reply:
Much more reliable approach :) Thanks!